base/strings/search.jl

```diff
@@ -3,18 +3,39 @@
 const Chars = Union{Char,Tuple{Vararg{Char}},AbstractVector{Char},Set{Char}}
 
 """
-    search(string::AbstractString, chars::Chars, [start::Integer])
+    search(string, chars, [start::Integer])
 
 Search for the first occurrence of the given characters within the given string. The second
 argument may be a single character, a vector or a set of characters, a string, or a regular
 expression (though regular expressions are only allowed on contiguous strings, such as ASCII
-or UTF-8 strings). The third argument optionally specifies a starting index. The return
-value is a range of indexes where the matching sequence is found, such that `s[search(s,x)] == x`:
+or UTF-8 strings).
+Alternatively the first and the second arguments may be arrays of `Int8` or `UInt8`.
```

Comment:
I'd rather not document this, it really should be an internal function which shouldn't have been exposed. I was preparing a PR to merge `search` with `findfirst` which would also handle this.